src/taipy/gui/version.json

```diff
@@ -1 +1 @@
-{"major": 1, "minor": 2, "patch": 0, "ext": "dev0"}
+{"major": 2, "minor": 0, "patch": 0, "ext": "dev0"}
```

Comment:
Good catch!

Reply:
this is what I mentioned during the planning but as I couldn't find a specific issue ...